src/lib.rs

```diff
@@ -113,10 +113,13 @@
 //! tracing_subscriber::fmt::init();
 //! ```
 //!
-//! And then run your application with, for example, `RUST_LOG=info` to get logs. If you also want
-//! automatic logging of errors, take a look at the documentation for
-//! [`color-eyre`](https://docs.rs/color_eyre/), which includes information about how to integrate
-//! with [`tracing-error`](https://docs.rs/tracing-error).
+//! And then run your application with, for example, `RUST_LOG=info` to get logs. If you're using
+//! the `log` crate, you can instead just add a dependency on `tracing` with the `log` feature
+//! enabled, and things should just "magically" work.
+//!
+//! If you also want automatic logging of errors (which I think you do), take a look at the
```

Comment:
tracing-error doesn't do automatic logging of errors. It just provides `SpanTrace` which gives automatic capture of spans with errors which it then prints in the report.

Reply:
Ah, sorry, what I meant to imply here (which, maybe I'm wrong?) was that `color_eyre` will create those `SpanTrace` for you whenever it propagates an error? But maybe I just completely made that up?

Reply:
you're not wrong, this is what `color_eyre` does, though I would say it does the capture when its converted, not when its propogated, though that's often the same thing. But I think the distinction is important because I'm pushing to add error return traces which would be specifically tied to propogation.

Reply:
If I'm understanding you correctly, I then still think this text is accurate? All it's saying is that _if_ you follow the `color_eyre` example, you get automatic logging of errors. Which it then sounds like is accurate. How do you think it should be rephrased?

Reply:
The confusion I'm worried about is people thinking that converting to a `color_eyre::Report` somehow triggers a tracing event which literally logs the error the moment it is constructed. Like, if you drop a report immediately after making it nothing gets logged.

I would probably just tweek it to "automatically include tracing spans with errors" or maybe "log tracing spans with your errors automatically" or something else along those lines. Just make it clear that it's not providing automatic _logging_.

Reply:
How's https://github.com/jonhoo/tsunami/pull/32/commits/80b29d0afd59935abb90f78c53310a0a7a5da947 ?

Reply:
looks perfect